# Icelandic translation for update-manager
# Copyright (c) 2006 Rosetta Contributors and Canonical Ltd 2006
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2009-10-09 08:51+0000\n"
"PO-Revision-Date: 2009-10-12 06:51+0000\n"
"Last-Translator: Baldur <baldurpet@gmail.com>\n"
"Language-Team: Icelandic <is@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2009-10-13 20:36+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:278 ../UpdateManager/Core/utils.py:278
msgid "0 KB"
msgstr "0 KB"

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:281 ../UpdateManager/Core/utils.py:281
msgid "1 KB"
msgstr "1 KB"

#. TRANSLATORS: download size of small updates, e.g. "250 KB"
#: ../DistUpgrade/utils.py:284 ../UpdateManager/Core/utils.py:284
#, python-format
msgid "%.0f KB"
msgstr "%.0f KB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:287 ../UpdateManager/Core/utils.py:287
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:208 ../DistUpgrade/distro.py:423
#, python-format
msgid "Server for %s"
msgstr "Netþjónn fyrir %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:226 ../DistUpgrade/distro.py:232
#: ../DistUpgrade/distro.py:248
msgid "Main server"
msgstr "Aðalnetþjónn"

#: ../DistUpgrade/distro.py:252
msgid "Custom servers"
msgstr "Sérsniðnir netþjónar"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:131
msgid "Could not calculate sources.list entry"
msgstr "Gat ekki reiknað út sources.list færslur"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:234
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Fann engar pakkaskrár, ertu viss um að þetta sé Ubuntu diskur? Þetta gæti "
"líka verið röng tölvuhögun."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:277
msgid "Failed to add the CD"
msgstr "Ekki tókst að bæta við geisladisknum"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:278
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Það kom babb í bátinn þegar geisladisknum var bætt við og því verður að "
"hætta við uppfærsluna. Tilkynntu þetta sem hugbúnaðarvillu ef þetta er "
"gildur Ubuntu geisladiskur.\n"
"\n"
"Villuskilaboðin voru:\n"
"‚%s‘"

#: ../DistUpgrade/DistUpgradeCache.py:130
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Fjarlægja pakka sem er í slæmu ásigkomulagi"
msgstr[1] "Fjarlægja þá pakka sem eru í slæmu ásigkomulagi"

#: ../DistUpgrade/DistUpgradeCache.py:133
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Pakkinn ‚%s‘ er í slæmu ásigkomulagi og það er mikilvægt að hann sé settur "
"upp aftur, en því miður fannst hann ekki í neinu gangasafni. Viltu eyða "
"pakkanum og halda áfram?"
msgstr[1] ""
"Pakkarnir ‚%s‘ eru í slæmu ásigkomulagi og það er mikilvægt er að þeir séu "
"settir upp aftur, en því miður fundust þeir ekki í neinu gangasafni. Viltu "
"eyða pökkunum og halda áfram?"

#: ../DistUpgrade/DistUpgradeCache.py:332
msgid "Broken packages"
msgstr "Bilaðir pakkar"

#: ../DistUpgrade/DistUpgradeCache.py:333
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Kerfið þitt inniheldur bilaða pakka sem ekki var hægt að laga með þessum "
"hugbúnaði. Lagaðu það með synaptic eða apt-get áður en lengra er haldið."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:609
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Villa kom upp við að reikna út uppfærsluna:\n"
"%s\n"
"\n"
" Nokkrar ástæður geta ollið þessu:\n"
" * Þú ert að uppfæra yfir í prufuútgáfu af Ubuntu\n"
" * Þú ert að nota prufuútgáfu af Ubuntu\n"
" * Þú hefur náð í hugbúnaðarpakka sem eru ekki á vegum Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:619
msgid "This is most likely a transient problem, please try again later."
msgstr "Þetta er væntanlega tímabundið vandamál. Reyndu aftur síðar."

#: ../DistUpgrade/DistUpgradeCache.py:622
msgid ""
"If none of this applies, then please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the bug "
"report."
msgstr ""
"Tilkynntu þessa villu um ‚update-manager‘ ef ekkert á þessu á við og láttu "
"fylgja afrit af /var/log/dist-upgrade/."

#: ../DistUpgrade/DistUpgradeCache.py:628
#: ../UpdateManager/UpdateManager.py:732
msgid "Could not calculate the upgrade"
msgstr "Gat ekki reiknað út uppfærsluna"

#: ../DistUpgrade/DistUpgradeCache.py:678
msgid "Error authenticating some packages"
msgstr "Villa við sannvottun einhverra pakka"

#: ../DistUpgrade/DistUpgradeCache.py:679
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Ekki tókst að sannvotta einhverja pakka. Þetta gæti verið skammvinnt "
"netvandamál. Þú gætir reynt aftur síðar. Sjá hér fyrir neðan fyrir lista "
"yfir pakka sem tókst að sannvotta."

#: ../DistUpgrade/DistUpgradeCache.py:699
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr "Það á að fjarlægja pakkann ‚%s‘ en það er bannað að fjarlægja hann."

#: ../DistUpgrade/DistUpgradeCache.py:703
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Það á að fjarlægja pakkann ‚%s‘ en hann er ómissandi."

#: ../DistUpgrade/DistUpgradeCache.py:712
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Reyni að setja upp útgáfuna '%s' sem er á svörtum lista"

#: ../DistUpgrade/DistUpgradeCache.py:812
#, python-format
msgid "Can not mark '%s' for upgrade"
msgstr "Ekki tókst að velja að uppfæra pakkann ‚%s‘"

#: ../DistUpgrade/DistUpgradeCache.py:826
#, python-format
msgid "Can't install '%s'"
msgstr "Ekki tókst að setja upp ‚%s‘"

#: ../DistUpgrade/DistUpgradeCache.py:827
msgid ""
"It was impossible to install a required package. Please report this as a "
"bug. "
msgstr ""
"Ekki tókst að setja upp þann pakka sem beðið var um. Tilkynntu þetta sem "
"hugbúnaðarvillu. "

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:836
msgid "Can't guess meta-package"
msgstr "Get ekki giskað á lýsi-pakka"

#: ../DistUpgrade/DistUpgradeCache.py:837
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Kerfið þitt inniheldur hvorki ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop né edubuntu-desktop- og ekki tókst að bera kennsl á hvaða tegund af "
"Ubuntu þú ert að nota.\n"
" Vinsamlegast settu upp einn af áðurnefndum pökkum með því að nota Synaptic "
"pakkastjórann eða með því að nota apt-get áður en þú heldur áfram."

#: ../DistUpgrade/DistUpgradeController.py:85
msgid "Reading cache"
msgstr "Les skyndiminni"

#: ../DistUpgrade/DistUpgradeController.py:181
msgid "Unable to get exclusive lock"
msgstr "Fékk ekki útilokunarlæsingu"

#: ../DistUpgrade/DistUpgradeController.py:182
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Þetta þýðir yfirleitt að annað pakkastjórnunarforrit (til dæmis ‚apt-get‘ "
"eða ‚aptitude‘) sé í gangi. Lokaðu því forriti fyrst."

#: ../DistUpgrade/DistUpgradeController.py:235
msgid "Upgrading over remote connection not supported"
msgstr "Uppfærsla yfir fjarskylda tengingu er ekki studd"

#: ../DistUpgrade/DistUpgradeController.py:236
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. The upgrade will abort now. Please try without "
"ssh."
msgstr ""
"Þú ert að uppfæra yfir fjarskulda ssh tengingu með frontenda sem styður "
"þetta ekki. Uppfærslan mun stöðvast núna.\r\n"
"Vinsamlegast reyndu án ssh"

#: ../DistUpgrade/DistUpgradeController.py:247
msgid "Continue running under SSH?"
msgstr "Viltu halda áfram að keyra í gegnum SSH?"

#: ../DistUpgrade/DistUpgradeController.py:248
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, a additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Þessi seta virðist keyra í gegnum ssh. Við mælum ekki með að uppfæra í "
"gegnum ssh sem stendur þar sem erfitt getur verið að halda áfram ef eitthvað "
"fer úrskeiðis.\n"
"\n"
"Ef þú heldur samt áfram verður nýr ssh þjónn stofnaður á gátt ‚%s‘.\n"
"Viltu halda áfram?"

#: ../DistUpgrade/DistUpgradeController.py:263
msgid "Starting additional sshd"
msgstr "Keyri annað sshd"

#: ../DistUpgrade/DistUpgradeController.py:264
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Til að auðvelda endurréttingu þá mun auka sshd keyra í tenginum ‚%s‘. Ef "
"eitthvað fer úrskeyðis við að keyra ssh þá er hægt að tengjast við hitt.\n"

#: ../DistUpgrade/DistUpgradeController.py:334
#: ../DistUpgrade/DistUpgradeController.py:379
msgid "Can not upgrade"
msgstr "Get ekki uppfært"

#: ../DistUpgrade/DistUpgradeController.py:335
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Þetta tól ræður ekki við uppfærslu frá ‚%s‘ til ‚%s'."

#: ../DistUpgrade/DistUpgradeController.py:344
msgid "Sandbox setup failed"
msgstr "Ekki tókst að setja upp sandkassann"

#: ../DistUpgrade/DistUpgradeController.py:345
msgid "It was not possible to create the sandbox environment."
msgstr "Það tókst ekki að búa til sandkassa."

#: ../DistUpgrade/DistUpgradeController.py:351
msgid "Sandbox mode"
msgstr "Sandkassahamur"

#: ../DistUpgrade/DistUpgradeController.py:352
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a systemdir from now until the next reboot are "
"permanent."
msgstr ""
"Þessi uppfærsla fer fram í sandkassanum (til að testa). Allar breytingar sem "
"þú gerir fara í ‚%s‘ og eyðast þegar þú kveikir næst á tölvunni.\n"
"\n"
"Allar breytingar sem gerðar eru á systemdir verða teknar aftur þegar þú "
"kveikir næst á tölvunni."

#: ../DistUpgrade/DistUpgradeController.py:380
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Uppsetning þín á python er skemmt. Vinsamlegast lagaðu ‚/usr/bin/python‘ "
"vísunina."

#: ../DistUpgrade/DistUpgradeController.py:406
msgid "Package 'debsig-verify' is installed"
msgstr "Pakkinn 'debsig-verify' uppsettur"

#: ../DistUpgrade/DistUpgradeController.py:407
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Uppfærslan getur ekki haldið áfram þar sem þessi pakki er uppsettur.\n"
"Vinsamlegast fjarlægðu hann með synaptic pakkastjóranum eða með því að nota "
"‚apt-get remove debsig-verify‘ fyrst og keyra svo uppfærsluna aftur."

#: ../DistUpgrade/DistUpgradeController.py:417
msgid "Include latest updates from the Internet?"
msgstr "Viltu láta nýjar uppfærslur af netinu fylgja með?"

#: ../DistUpgrade/DistUpgradeController.py:418
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Uppfærslukerfið getur notað netið til að sækja nýjar uppfærslur sjálfkrafa "
"og að setja þær upp á meðan á uppfærslunni stendur. Þetta er ráðlagt ef þú "
"hefur nettengingu.\n"
"\n"
"Það tekur lengri tíma ef þú sækir uppfærslur af netinu, en á móti verður "
"tölvan af nýjustu gerð að uppfærslunni lokinni. Þú getur valið um að gera "
"þetta ekki en þú ættir að uppfæra kerfið þegar samt sem áður.\n"
"Netið verður ekki notað ef þú velur að gera þetta ekki."

#: ../DistUpgrade/DistUpgradeController.py:641
#, python-format
msgid "disabled on upgrade to %s"
msgstr "óvirkt að uppfæra til %s"

#: ../DistUpgrade/DistUpgradeController.py:667
msgid "No valid mirror found"
msgstr "Enginn spegill fannst sem virkar"

#: ../DistUpgrade/DistUpgradeController.py:668
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:688
msgid "Generate default sources?"
msgstr "Viltu búa til uppsprettur?"

#: ../DistUpgrade/DistUpgradeController.py:689
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:724
msgid "Repository information invalid"
msgstr "Ógildar útibúaupplýsingar"

#: ../DistUpgrade/DistUpgradeController.py:725
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug."
msgstr ""
"Villa kom upp við að uppfæra upplýsingar um gagnahirslu. Taktu þér tíma við "
"að tilkynna þessa villu ef þú nennir."

#: ../DistUpgrade/DistUpgradeController.py:731
msgid "Third party sources disabled"
msgstr "Frumgögn frá þriðja aðila voru gerðir óvirkir"

#: ../DistUpgrade/DistUpgradeController.py:732
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Einhverjir þriðju aðilar voru gerðir óvirkir í skránni sources.list. Þú "
"getur virkjað þá seinna með forritinu ‚software-properties‘ eða með "
"pakkastjóranum þínum."

#: ../DistUpgrade/DistUpgradeController.py:772
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Pakkinn er eitthvað ójafn"
msgstr[1] "Pakkarnir eru eitthvað ójafn"

#: ../DistUpgrade/DistUpgradeController.py:775
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Það þarf að setja pakkann ‚%s‘ upp aftur vegna þess að hann er í einhverju "
"ójafnvægi, en hann fannst ekki í geymslunni. Þú verður annað hvort að setja "
"hann upp í höndunum eða fjarlægja hann af tölvunni."
msgstr[1] ""
"Það þarf að setja pakkana ‚%s‘ upp aftur vegna þess að þeir eru í einhverju "
"ójafnvægi, en þeir fundust ekki í geymslunni. Þú verður annað hvort að setja "
"þá upp í höndunum eða fjarlægja þá af tölvunni."

#: ../DistUpgrade/DistUpgradeController.py:823
msgid "Error during update"
msgstr "Villa kom upp við að uppfæra tölvuna"

#: ../DistUpgrade/DistUpgradeController.py:824
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Villa kom upp við að uppfæra tölvuna. Þetta er oftast villa við nettenginuna "
"þannig best væri að kíkja á hana og reyna aftur."

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:833
#: ../UpdateManager/UpdateManager.py:555
msgid "Not enough free disk space"
msgstr "Það vantar meira pláss á diskinum"

#: ../DistUpgrade/DistUpgradeController.py:834
#, python-format
msgid ""
"The upgrade is now aborted. The upgrade needs a total of %s free space on "
"disk '%s'. Please free at least an additional %s of disk space on '%s'. "
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""
"Uppfærslan hætti vegna þess að hún þarf %s af plássi á diskinum ‚%s‘. Þú "
"þarft sem sagt að útvega %s af plássi á ‚%s‘. Það gæti verið sniðugt að tæmi "
"ruslið eða að henda gömlum pökkum með því að nota skipunina ‚sudo apt-get "
"clean‘."

#: ../DistUpgrade/DistUpgradeController.py:874
msgid "Support for some applications ended"
msgstr "Sum forrit eru ekki lengur studd"

#: ../DistUpgrade/DistUpgradeController.py:875
msgid ""
"Canonical Ltd. no longer provides support for the following software "
"packages. You can still get support from the community.\n"
"\n"
"If you have not enabled community maintained software (universe), these "
"packages will be suggested for removal at the end of the upgrade."
msgstr ""
"Canonical Ltd. styður ekki lengur eftirfarandi forrit. Þessu er þó ennþá "
"viðhaldið af samfélaginu.\n"
"\n"
"Ef þú hefur ekki leyft hugbúnað sem viðhaldið er af samfélaginu, þá verður "
"boðist til að eyða þessum pökkum eftir uppfærsluna."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:884
#: ../DistUpgrade/DistUpgradeController.py:1554
msgid "Calculating the changes"
msgstr "Reikna út breytingar"

#. ask the user if he wants to do the changes
#: ../DistUpgrade/DistUpgradeController.py:899
msgid "Do you want to start the upgrade?"
msgstr "Viltu byrja að uppfæra tölvuna?"

#: ../DistUpgrade/DistUpgradeController.py:955
msgid "Upgrade canceled"
msgstr "Hætt við uppfærslu"

#: ../DistUpgrade/DistUpgradeController.py:956
msgid ""
"The upgrade cancels now and the original system state is restored. You can "
"resume the upgrade at a later time."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:961
#: ../DistUpgrade/DistUpgradeController.py:1048
msgid "Could not download the upgrades"
msgstr "Gat ekki sótt uppfærslurnar"

#: ../DistUpgrade/DistUpgradeController.py:962
msgid ""
"The upgrade is now aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far are kept."
msgstr ""
"Uppfærslan hætti. Athugaðu hvort það sé ekki allt í lagi með nettenginguna "
"þína og reyndu aftur. Þær skrár sem er þegar búið að sækja verður haldið."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1015
#: ../DistUpgrade/DistUpgradeController.py:1138
msgid "Error during commit"
msgstr "Villa við staðfestingu"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1017
#: ../DistUpgrade/DistUpgradeController.py:1177
msgid "Restoring original system state"
msgstr "Endurstilli stýrikerfi í upprunalegt form"

#: ../DistUpgrade/DistUpgradeController.py:1018
#: ../DistUpgrade/DistUpgradeController.py:1032
msgid "Could not install the upgrades"
msgstr "Gat ekki sett upp uppfærslurnar"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1023
msgid ""
"The upgrade is now aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Uppfærslan hætti, og það er mögulegt að kerfið sé ónothæft. Tölvan reynir að "
"redda málunum (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1028
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"files in /var/log/dist-upgrade/ in the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Sendu villuskýrslu út af pakkanum ‚update-manager‘-manager' og láttu "
"skrárnar í /var/log/dist-upgrade/ fylgja með.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1049
msgid ""
"The upgrade is now aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Uppfærslan hætti. Athugaðu hvort það sé ekki allt í lagi með nettenginguna "
"og reyndu aftur. "

#: ../DistUpgrade/DistUpgradeController.py:1126
msgid "Remove obsolete packages?"
msgstr "Viltu fjarlægja úrelta pakka?"

#: ../DistUpgrade/DistUpgradeController.py:1127
#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "_Keep"
msgstr "_Halda"

#: ../DistUpgrade/DistUpgradeController.py:1127
msgid "_Remove"
msgstr "_Fjarlægja"

#: ../DistUpgrade/DistUpgradeController.py:1139
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Það kom upp vandamál við að hreinsa til. Lestu skilaboðin fyrir neðan til að "
"fá frekari upplýsingar. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1215
msgid "Required depends is not installed"
msgstr "Það er ekki búið að setja upp nauðsynleg ákvæði"

#: ../DistUpgrade/DistUpgradeController.py:1216
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Það er ekki búið að setja upp nauðsynlega ákvæðið ‚%s‘. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1461
#: ../DistUpgrade/DistUpgradeController.py:1528
msgid "Checking package manager"
msgstr "Athuga með pakkastjóra"

#: ../DistUpgrade/DistUpgradeController.py:1466
msgid "Preparing the upgrade failed"
msgstr "Undirbúningur fyrir uppfærslu mistókst"

#: ../DistUpgrade/DistUpgradeController.py:1467
msgid ""
"Preparing the system for the upgrade failed. Please report this as a bug "
"against the 'update-manager' package and include the files in /var/log/dist-"
"upgrade/ in the bug report."
msgstr ""
"Villa kom upp við að undirbúa kerfið undir uppfærslu. Tilkynntu um þetta sem "
"villu í pakkanum ‚update-manager‘ og láttu skrárnar í /var/log/dis-upgrade/ "
"fylgja með."

#: ../DistUpgrade/DistUpgradeController.py:1483
msgid "Getting upgrade prerequisites failed"
msgstr "Það tókst ekki að útvega forsendu fyrir uppfærsluna"

#: ../DistUpgrade/DistUpgradeController.py:1484
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug against the 'update-manager' package and include "
"the files in /var/log/dist-upgrade/ in the bug report."
msgstr ""
"Kerfið gat ekki útvegað forsendu fyrir uppfærsluna. Hún hættir núna og "
"kerfið verður eins og það var.\n"
"\n"
"Tilkynntu um þetta sem villu í pakkanum ‚update-manager‘ og láttu skrárnar í "
"/var/log/dis-upgrade/ fylgja með."

#: ../DistUpgrade/DistUpgradeController.py:1514
msgid "Updating repository information"
msgstr "Uppfæri upplýsingar um gagnahirslur"

#: ../DistUpgrade/DistUpgradeController.py:1542
msgid "Invalid package information"
msgstr "Ógildar pakkaupplýsingar"

#: ../DistUpgrade/DistUpgradeController.py:1543
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the bug "
"report."
msgstr ""
"Pakkinn ‚%s‘ sem er mjög mikilvægur, fannst ekki eftir að þú uppfærðir "
"pakkaupplýsingarnar þínar.\n"
"Þetta er grafalvarleg villa, tilkynntu um að það sé villa í pakkanum ‚update-"
"manager‘ og láttu skrárnar í /var/log/dis-upgrade/ fylgja með."

#: ../DistUpgrade/DistUpgradeController.py:1560
#: ../DistUpgrade/DistUpgradeController.py:1608
msgid "Fetching"
msgstr "Sæki"

#: ../DistUpgrade/DistUpgradeController.py:1566
#: ../DistUpgrade/DistUpgradeController.py:1612
msgid "Upgrading"
msgstr "Uppfæri"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1571
#: ../DistUpgrade/DistUpgradeController.py:1614
#: ../DistUpgrade/DistUpgradeController.py:1621
#: ../DistUpgrade/DistUpgradeController.py:1626
msgid "Upgrade complete"
msgstr "Uppfærslu lokið"

#: ../DistUpgrade/DistUpgradeController.py:1572
#: ../DistUpgrade/DistUpgradeController.py:1615
#: ../DistUpgrade/DistUpgradeController.py:1622
msgid ""
"The upgrade is completed but there were errors during the upgrade process."
msgstr "Uppfærslunni er lokið, en það komu upp nokkrar villur."

#: ../DistUpgrade/DistUpgradeController.py:1579
msgid "Searching for obsolete software"
msgstr "Leita að úreltum hugbúnaði"

#: ../DistUpgrade/DistUpgradeController.py:1584
msgid "System upgrade is complete."
msgstr "Uppfærslu á stýrikerfi er lokið."

#: ../DistUpgrade/DistUpgradeController.py:1627
msgid "The partial upgrade was completed."
msgstr "Takmarkaðri uppfærslu er lokið."

#: ../DistUpgrade/DistUpgradeQuirks.py:148
msgid "No ARMv6 CPU"
msgstr "Enginn ARMv6 örgjörvi"

#: ../DistUpgrade/DistUpgradeQuirks.py:149
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""

#. TRANSLATORS: this string is not displayed in the 9.10 upgrade
#. (but it will be used for 8.04 -> 10.04 LTS upgrades)
#: ../DistUpgrade/DistUpgradeQuirks.py:169
#: ../DistUpgrade/DistUpgradeQuirks.py:253
#: ../DistUpgrade/DistUpgradeQuirks.py:280
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Það að uppfæra getur valdið því að brellur, leikir og önnur forrit sem "
"krefjast mikils af skjákortinu verði ekki eins flott."

#: ../DistUpgrade/DistUpgradeQuirks.py:173
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04.\n"
"\n"
"Do you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:257
#: ../DistUpgrade/DistUpgradeQuirks.py:284
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04.\n"
"\n"
"Do you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:454
msgid "evms in use"
msgstr "evms í notkun"

#: ../DistUpgrade/DistUpgradeQuirks.py:455
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:43
msgid "Sandbox upgrade using aufs"
msgstr "Uppfærsla í sandkassanum með aufs"

#: ../DistUpgrade/DistUpgradeMain.py:45
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Nota þessa slóð til að leita að geisladiski með pökkum sem hægt er að "
"uppfæra með"

#: ../DistUpgrade/DistUpgradeMain.py:51
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Nota framenda. Hægt er að nálgast: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:54
msgid "*DEPRECATED* this option will be ignore"
msgstr "*ÓÆSKILEGT* þessi valmöguleiki verður hunsaður"

#: ../DistUpgrade/DistUpgradeMain.py:57
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "Ekki uppfæra allt (mun ekki skrifa yfir sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:59
msgid "Set datadir"
msgstr "Ákveða datadir"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:106
#: ../DistUpgrade/DistUpgradeViewKDE.py:190
#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Settu ‚%s‘ í drif ‚%s‘"

#: ../DistUpgrade/DistUpgradeViewGtk.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:203
msgid "Fetching is complete"
msgstr "Búið að sækja"

#: ../DistUpgrade/DistUpgradeViewGtk.py:137
#: ../DistUpgrade/DistUpgradeViewKDE.py:216
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Sæki skránna %li af %li á %sB/sek"

#: ../DistUpgrade/DistUpgradeViewGtk.py:138
#: ../DistUpgrade/DistUpgradeViewGtk.py:273
#: ../DistUpgrade/DistUpgradeViewKDE.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:357
#, python-format
msgid "About %s remaining"
msgstr "Um það bil %s eftir"

#: ../DistUpgrade/DistUpgradeViewGtk.py:140
#: ../DistUpgrade/DistUpgradeViewKDE.py:219
#, python-format
msgid "Fetching file %li of %li"
msgstr "Sæki skrá %li af %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:172
#: ../DistUpgrade/DistUpgradeViewKDE.py:253
msgid "Applying changes"
msgstr "Virkja breytingar"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:197
#: ../DistUpgrade/DistUpgradeViewKDE.py:266
msgid "dependency problems - leaving unconfigured"
msgstr "afleiðingavilla - skil eftir óstillt"

#: ../DistUpgrade/DistUpgradeViewGtk.py:202
#: ../DistUpgrade/DistUpgradeViewKDE.py:268
#, python-format
msgid "Could not install '%s'"
msgstr "Gat ekki sett upp ‚%s‘"

#: ../DistUpgrade/DistUpgradeViewGtk.py:203
#: ../DistUpgrade/DistUpgradeViewKDE.py:269
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:219
#: ../DistUpgrade/DistUpgradeViewKDE.py:290
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Viltu skipta út stillingarskránni sem þú hefur breytt\n"
"‚%s‘?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:220
#: ../DistUpgrade/DistUpgradeViewKDE.py:291
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Þú munt tapa öllum breytingum sem þú hefur gert við þessa stillingarskrá ef "
"þú ákveður að skipta henni út fyrir nýja útgáfu."

#: ../DistUpgrade/DistUpgradeViewGtk.py:233
#: ../DistUpgrade/DistUpgradeViewKDE.py:309
msgid "The 'diff' command was not found"
msgstr "Skipunin ‚diff‘‘ fannst ekki"

#: ../DistUpgrade/DistUpgradeViewGtk.py:419
#: ../DistUpgrade/DistUpgradeViewText.py:107
msgid "A fatal error occurred"
msgstr "Það kom upp grafalvarleg villa"

#: ../DistUpgrade/DistUpgradeViewGtk.py:420
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade is now aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk.py:435
msgid "Ctrl-c pressed"
msgstr "Ýtt á ctrl-c"

#: ../DistUpgrade/DistUpgradeViewGtk.py:436
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Þetta mun stöðva aðgerðina og skilja kerfið eftir bilað. Ertu viss um að þú "
"viljir halda áfram?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:578
msgid "To prevent data loss close all open applications and documents."
msgstr "Til að fyrirbyggja tap á gögnum skaltu loka öllum forritum og skrám."

#: ../DistUpgrade/DistUpgradeViewGtk.py:592
#, python-format
msgid "<b>Downgrade %s</b>"
msgstr "<b>Lækka %s</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:594
#: ../DistUpgrade/DistUpgradeViewKDE.py:757
#, python-format
msgid "Remove %s"
msgstr "Fjarlægja %s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:599
#: ../DistUpgrade/DistUpgradeViewKDE.py:759
#, python-format
msgid "Install %s"
msgstr "Setja %s upp"

#: ../DistUpgrade/DistUpgradeViewGtk.py:601
#: ../DistUpgrade/DistUpgradeViewKDE.py:761
#, python-format
msgid "Upgrade %s"
msgstr "Uppfæri %s"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:191
#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
msgid "Media Change"
msgstr "Breyta miðli"

#: ../DistUpgrade/DistUpgradeViewKDE.py:321
msgid "Show Difference >>>"
msgstr "Sýna mun >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:324
msgid "<<< Hide Difference"
msgstr "<<< Fela mun"

#: ../DistUpgrade/DistUpgradeViewKDE.py:538
msgid "Error"
msgstr "Villa"

#: ../DistUpgrade/DistUpgradeViewKDE.py:550
msgid "&Cancel"
msgstr "&Hætta við"

#: ../DistUpgrade/DistUpgradeViewKDE.py:552
#: ../DistUpgrade/DistUpgradeViewKDE.py:789
msgid "&Close"
msgstr "&Loka"

#: ../DistUpgrade/DistUpgradeViewKDE.py:598
msgid "Show Terminal >>>"
msgstr "Sýna útstöðina >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:601
msgid "<<< Hide Terminal"
msgstr "<<< Fela útstöðina"

#: ../DistUpgrade/DistUpgradeViewKDE.py:681
msgid "Information"
msgstr "Upplýsingar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:731
#: ../DistUpgrade/DistUpgradeViewKDE.py:772
#: ../DistUpgrade/DistUpgradeViewKDE.py:775 ../DistUpgrade/DistUpgrade.ui.h:9
msgid "Details"
msgstr "Upplýsingar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:785
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "Restart required"
msgstr "Þú þarft að endurræsa tölvuna"

#: ../DistUpgrade/DistUpgradeViewKDE.py:785 ../DistUpgrade/DistUpgrade.ui.h:5
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Endurræsa kerfið til að ljúka uppfærslunni</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:788 ../DistUpgrade/DistUpgrade.ui.h:23
msgid "_Restart Now"
msgstr "_Endurræsa núna"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:800
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Viltu hætta við uppfærsluna?</big></b>\n"
"\n"
"Það getur verið að tölvan þín verði ónothæf ef þú hættir núna. Það er "
"stranglega mælt með því að halda áfram."

#: ../DistUpgrade/DistUpgradeViewKDE.py:804
msgid "Cancel Upgrade?"
msgstr "Viltu hætta við að uppfæra?"

#: ../DistUpgrade/DistUpgradeView.py:59
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li dagur"
msgstr[1] "%li dagar"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li klukkutími"
msgstr[1] "%li klukkutímar"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li mínúta"
msgstr[1] "%li mínútur"

#: ../DistUpgrade/DistUpgradeView.py:64
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li sekúnda"
msgstr[1] "%li sekúndur"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:77
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:92
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:113
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr "Þetta tekur um %s með 1Mbit DSL tengingu og um %s með 56k módemi."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:117
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Þetta mun taka um %s með þinni tengingu. "

#: ../DistUpgrade/DistUpgradeView.py:186 ../DistUpgrade/DistUpgrade.ui.h:14
msgid "Preparing to upgrade"
msgstr "Undirbý að uppfæra"

#: ../DistUpgrade/DistUpgradeView.py:187
msgid "Getting new software channels"
msgstr "Sæki nýjar hugbúnaðarrásir"

#: ../DistUpgrade/DistUpgradeView.py:188 ../DistUpgrade/DistUpgrade.ui.h:12
msgid "Getting new packages"
msgstr "Sæki nýja pakka"

#: ../DistUpgrade/DistUpgradeView.py:189 ../DistUpgrade/DistUpgrade.ui.h:13
msgid "Installing the upgrades"
msgstr "Set upp uppfærslurnar"

#: ../DistUpgrade/DistUpgradeView.py:190 ../DistUpgrade/DistUpgrade.ui.h:8
msgid "Cleaning up"
msgstr "Hreinsa til"

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:263
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d pakki mun vera fjarlægður."
msgstr[1] "%d pakkar munu vera fjarlægðir."

#: ../DistUpgrade/DistUpgradeView.py:268
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d nýr pakki mun vera uppsettur."
msgstr[1] "%d nýir pakkar munu vera uppsettir."

#: ../DistUpgrade/DistUpgradeView.py:274
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d pakki mun vera uppfærður."
msgstr[1] "%d pakkar munu vera uppfærðir."

#: ../DistUpgrade/DistUpgradeView.py:279
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Þú þarft að sækja samtals %s. "

#: ../DistUpgrade/DistUpgradeView.py:283
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be cancelled."
msgstr ""
"Það getur tekið nokkra tíma að uppfæra tölvuna. Ekki er hægt að hætta við "
"þegar niðurhalinu er lokið."

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:289 ../UpdateManager/UpdateManager.py:489
msgid "Your system is up-to-date"
msgstr "Kerfið þitt er uppfært að fullu"

#: ../DistUpgrade/DistUpgradeView.py:290
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Það eru engar uppfærslur fáanlega fyrir kerfið þitt. Uppfærslu verður nú "
"hætt."

#: ../DistUpgrade/DistUpgradeView.py:303
msgid "Reboot required"
msgstr "Þú þarft að endurræsa tölvuna"

#: ../DistUpgrade/DistUpgradeView.py:304
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Búið er að uppfæra tölvuna þína og nú þarft þú að endurræsa hana. Viltu gera "
"það núna?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:134
#: ../DistUpgrade/DistUpgradeFetcherCore.py:135
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:134
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:135
msgid "Could not run the upgrade tool"
msgstr "Gat ekki keyrt uppfærsluforritið"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:135
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:135
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug"
msgstr ""
"Það er sennilega villa í uppfærsluforritinu. Tilkynntu þetta sem villu."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:204
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:204
msgid "Upgrade tool signature"
msgstr "Auðkenni uppfærsluforritsins"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:207
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:207
msgid "Upgrade tool"
msgstr "Uppfærsluforrit"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:237
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:237
msgid "Failed to fetch"
msgstr "Ekki tókst að sækja"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:238
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Það tókst ekki að sækja uppfærsluna. Þetta gæti verið vandamál með netið. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:242
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:242
msgid "Failed to extract"
msgstr "Mistókst að ná í upplýsingarnar"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:243
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Það mistókst að nálgast upplýsingarnar um uppfærsluna. Þetta er mögulega "
"vandamál við nettenginguna eða netþjóninn. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:248
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:248
msgid "Verification failed"
msgstr "Sannreynsla mistóks"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:249
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Það tókst ekki að sannprófa uppfærsluna. Þetta er mögulega vandamál við "
"nettenginguna eða netþjóninn. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:254
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:254
msgid "Authentication failed"
msgstr "Auðkenning mistókst"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:255
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Það tókst ekki að auðkenna uppfærsluna. Þetta er mögulega vandamál við "
"nettenginguna eða netþjóninn. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:263
#: ../DistUpgrade/DistUpgradeFetcherCore.py:269
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:263
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:269
msgid "Can not run the upgrade"
msgstr "Get ekki keyrt uppfærsluna"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:264
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:264
msgid ""
"This usually is caused by a system were /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:270
#, python-format
msgid "The error message is '%s'."
msgstr "Villumeldingin er ‚%s‘."

#: ../DistUpgrade/DistUpgradeViewText.py:108
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade is now aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:132
msgid "Aborting"
msgstr "Hætti við"

#: ../DistUpgrade/DistUpgradeViewText.py:137
msgid "Demoted:\n"
msgstr "Lækkað í tign:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:169
#: ../DistUpgrade/DistUpgradeViewText.py:191
#: ../DistUpgrade/DistUpgradeViewText.py:198
msgid "Continue [yN] "
msgstr "Halda áfram [jN] "

#: ../DistUpgrade/DistUpgradeViewText.py:169
#: ../DistUpgrade/DistUpgradeViewText.py:191
msgid "Details [d]"
msgstr "Upplýsingar [u]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:173
#: ../DistUpgrade/DistUpgradeViewText.py:201
msgid "y"
msgstr "j"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:176
#: ../DistUpgrade/DistUpgradeViewText.py:208
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:179
msgid "d"
msgstr "u"

#: ../DistUpgrade/DistUpgradeViewText.py:183
#, python-format
msgid "Remove: %s\n"
msgstr "Fjarlægja: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:186
#, python-format
msgid "Install: %s\n"
msgstr "Setja upp: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:189
#, python-format
msgid "Upgrade: %s\n"
msgstr "Uppfæra: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:205
msgid "Continue [Yn] "
msgstr "Halda áfram [Jn] "

#: ../DistUpgrade/DistUpgradeViewText.py:226
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Þú þarft að endurræsa tölvuna til að ljúka uppfærslunni.\n"
"Ef þú velur ‚j‘ mun hún endurræsa sig."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Viltu hætta við uppfærsluna sem er þegar hafin?</big></b>\n"
"\n"
"Kerfið gæti orðið ónothæft ef þú hættir við uppfærlsuna núna. Það er "
"sterklega mælt með að þú haldir áfram með uppfærsluna."

#: ../DistUpgrade/DistUpgrade.ui.h:6
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Viltu byrja að uppfæra?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:7
msgid "<b><big>Upgrading Ubuntu to version 9.10</big></b>"
msgstr "<b><big>Uppfæri í útgáfu 9.10 af Ubuntu</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Breytinga á milli skráa"

#: ../DistUpgrade/DistUpgrade.ui.h:11
msgid "Distribution Upgrade"
msgstr "Uppfæra kerfið"

#: ../DistUpgrade/DistUpgrade.ui.h:15
msgid "Restarting the computer"
msgstr "Endurræsi tölvuna"

#: ../DistUpgrade/DistUpgrade.ui.h:16
msgid "Setting new software channels"
msgstr "Ákveð nýjar hugbúnaðarrásir"

#: ../DistUpgrade/DistUpgrade.ui.h:17
msgid "Terminal"
msgstr "Útstöð"

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "_Cancel Upgrade"
msgstr "_Hætta við"

#: ../DistUpgrade/DistUpgrade.ui.h:19
msgid "_Continue"
msgstr "_Halda áfram"

#: ../DistUpgrade/DistUpgrade.ui.h:21
msgid "_Replace"
msgstr "_Skipta út"

#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "_Report Bug"
msgstr "_Tilkynna um villu"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "_Resume Upgrade"
msgstr "Hefja _aftur uppfærslu"

#: ../DistUpgrade/DistUpgrade.ui.h:25
msgid "_Start Upgrade"
msgstr "_Byrja að uppfæra"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:42
msgid "Please wait, this can take some time."
msgstr "Hinkraðu augnablik, þetta gæti tekið drjúga stund."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:44
msgid "Update is complete"
msgstr "Uppfærslu er lokið"

#: ../UpdateManager/DistUpgradeFetcher.py:85
#: ../UpdateManager/DistUpgradeFetcherKDE.py:103
msgid "Could not find the release notes"
msgstr "Fann ekki upplýsingar um útgáfu"

#: ../UpdateManager/DistUpgradeFetcher.py:86
#: ../UpdateManager/DistUpgradeFetcherKDE.py:104
msgid "The server may be overloaded. "
msgstr "Það er mögulega mikið álag á netþjóninum. "

#: ../UpdateManager/DistUpgradeFetcher.py:96
#: ../UpdateManager/DistUpgradeFetcherKDE.py:108
msgid "Could not download the release notes"
msgstr "Gat ekki sótt upplýsingar um útgáfuna"

#: ../UpdateManager/DistUpgradeFetcher.py:97
#: ../UpdateManager/DistUpgradeFetcherKDE.py:109
msgid "Please check your internet connection."
msgstr "Athugaðu hvort það sé allt í lagi með nettenginguna þína."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:62
#: ../UpdateManager/DistUpgradeFetcherKDE.py:85
msgid "Upgrade"
msgstr "Uppfæra"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:89
#: ../data/glade/UpdateManager.ui.h:12
msgid "Release Notes"
msgstr "Upplýsingar"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:128
#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
msgid "Downloading additional package files..."
msgstr "Sæki aukapakka..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Skrá %s af %s á %sB/sek"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
#, python-format
msgid "File %s of %s"
msgstr "Skrá %s af %s"

#: ../UpdateManager/ChangelogViewer.py:70
msgid "Open Link in Browser"
msgstr "Opna tengil í netvafra"

#: ../UpdateManager/ChangelogViewer.py:72
msgid "Copy Link to Clipboard"
msgstr "Afrita tengil"

#: ../UpdateManager/GtkProgress.py:115
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Sæki skrá %(current)li af %(total)li á %(speed)s/sek"

#: ../UpdateManager/GtkProgress.py:120
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Sæki skrá %(current)li af %(total)li"

#: ../UpdateManager/UpdateManager.py:302
#, python-format
msgid "Version %s: \n"
msgstr "Útgáfa %s: \n"

#: ../UpdateManager/UpdateManager.py:359
msgid "Downloading list of changes..."
msgstr "Sæki lista yfir breytingar..."

#: ../UpdateManager/UpdateManager.py:399
msgid "_Uncheck All"
msgstr "_Afvelja allt"

#: ../UpdateManager/UpdateManager.py:405
msgid "_Check All"
msgstr "_Velja allt"

#: ../UpdateManager/UpdateManager.py:449
#, python-format
msgid "%s selected. Download size: %s"
msgstr "%s valið. Stærð: %s"

#: ../UpdateManager/UpdateManager.py:454
msgid "Unknown download size"
msgstr "Stærð niðurhals er ekki þekkt"

#: ../UpdateManager/UpdateManager.py:462
msgid ""
"It is unknown when the package information was updated last. Please try "
"clicking on the 'Check' button to update the information."
msgstr ""
"Það er ekki vitað hvenær pakkaupplýsingarnar voru uppfærðar síðast. Ýttu á "
"‚kanna‘ takkann til að sækja nýjar upplýsingar."

#: ../UpdateManager/UpdateManager.py:471
#, python-format
msgid "The package information was last updated %s day ago."
msgid_plural "The package information was last updated %s days ago."
msgstr[0] "Pakkaupplýsingar voru uppfærðar fyrir %s degi."
msgstr[1] "Pakkaupplýsingar voru uppfærðar fyrir %s dögum."

#: ../UpdateManager/UpdateManager.py:475
#, python-format
msgid "The package information was last updated %s hour ago."
msgid_plural "The package information was last updated %s hours ago."
msgstr[0] "Pakkaupplýsingar voru uppfærðar fyrir %s tíma."
msgstr[1] "Pakkaupplýsingar voru uppfærðar fyrir %s tímum."

#: ../UpdateManager/UpdateManager.py:479
msgid "The package information was last updated less than one hour ago."
msgstr "Pakkaupplýsingarnar voru uppfærðar fyrir minna en klukkustund."

#: ../UpdateManager/UpdateManager.py:487 ../data/glade/UpdateManager.ui.h:22
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Hugbúnaðauppfærslur leiðrétta villur, lagfæra öryggisveilur og innleiða nýja "
"möguleika."

#: ../UpdateManager/UpdateManager.py:503
msgid "Welcome to Ubuntu"
msgstr "Velkomin í Ubuntu"

#: ../UpdateManager/UpdateManager.py:504
msgid ""
"These software updates have been issued since Ubuntu was released. If you "
"don't want to install them now, choose \"Update Manager\" from the "
"Administration Menu later."
msgstr ""
"Þessar uppfærslur voru gefnar út eftir að Ubuntu var gefið út. Ef þú vilt "
"ekki setja þær upp núna getur þú farið í ‚Uppfærslustjórann‘ í "
"aðalvalmyndinni (undir ‚Stjórnun‘) síðar."

#: ../UpdateManager/UpdateManager.py:507
msgid "Software updates are available for this computer"
msgstr "Þú getur náð í hugbúnaðaruppfærslur fyrir þessa tölvu"

#: ../UpdateManager/UpdateManager.py:508
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from the "
"Administration menu later."
msgstr ""
"Þú getur farið í „Uppfærslustjórann“ í stjórnunarvalmyndinni seinna ef þú "
"vilt ekki setja þær upp núna."

#: ../UpdateManager/UpdateManager.py:556
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"Uppfærslan krefst %s af plássi á diskinum ‚%s‘. Reyndu að rýma til alla "
"vegana %s af plássi á ‚%s‘. Prufaðu að tæma ruslið og að fjarlægja "
"bráðabirgðapakka sem gamlar uppsetningar skildu eftir sig með því að skrifa "
"‚sudo apt-get clean‘ í útstöðina."

#: ../UpdateManager/UpdateManager.py:593
#: ../UpdateManagerHildon/UpdateManagerHildon.py:114
msgid "Reading package information"
msgstr "Les pakkaupplýsingar"

#: ../UpdateManager/UpdateManager.py:710
msgid "Could not initialize the package information"
msgstr ""

#: ../UpdateManager/UpdateManager.py:711
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""

#: ../UpdateManager/UpdateManager.py:733
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""

#: ../UpdateManager/UpdateManager.py:756
msgid " (New install)"
msgstr " (ný uppsetning)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:760
#, python-format
msgid "(Size: %s)"
msgstr "(Stærð: %s)"

#: ../UpdateManager/UpdateManager.py:762
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Frá útgáfu %(old_version)s yfir í %(new_version)s"

#: ../UpdateManager/UpdateManager.py:766
#, python-format
msgid "Version %s"
msgstr "Útgáfa %s"

#: ../UpdateManager/UpdateManager.py:779
msgid "Your Ubuntu release is not supported anymore"
msgstr "Ubuntu útgáfan þín hefur ekki stuðning lengur"

#: ../UpdateManager/UpdateManager.py:780
msgid ""
"You will not get any further security fixes or critical updates. Please "
"Upgrade to a later version of Ubuntu Linux."
msgstr ""
"Það verða ekki frekari öryggisuppfærslur eða aðrar nauðsynlegar uppfærslur. "
"Vinsamlegast uppfærið í nýlegri útgáfu af Ubuntu GNU/Linux."

#: ../UpdateManager/UpdateManager.py:787
msgid "Upgrade information"
msgstr "Upplýsingar um uppfærslu"

#: ../UpdateManager/UpdateManager.py:814
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Nýa '%s' Ubuntu útgáfan er fáanleg</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:853
msgid "Software index is broken"
msgstr "Hugbúnaðarvísir er bilaður"

#: ../UpdateManager/UpdateManager.py:854
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Það er ómögulegt að setja upp eða fjarlægja nokkurn hugbúnað. Vinsamlegast "
"notaðu pakkastjórann „Synaptic‟ eða keyrðu skipunina „sudo apt-get install -"
"f‟ í útstöðina til að laga þetta vandamál."

#: ../UpdateManagerHildon/UpdateManagerHildon.py:136
msgid "Downloading Package Information"
msgstr "Hleð niður pakkaupplýsingum"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:137
msgid ""
"The repositories are being checked for new, removed, or updated software "
"packages"
msgstr ""

#: ../UpdateManagerHildon/UpdateManagerHildon.py:145
msgid "Downloading Package Updates"
msgstr "Sæki pakkaupplýsingar"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:146
msgid ""
"The selected package updates are being downloaded and installed on the system"
msgstr "Verið er að sækja og setja upp þær pakkaupplýsingar sem þú valdir"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:152
msgid "run_synaptic called with unknown action"
msgstr ""

#: ../UpdateManagerText/UpdateManagerText.py:26
msgid "Cancel"
msgstr "Hætta við"

#: ../UpdateManagerText/UpdateManagerText.py:27
msgid "Install"
msgstr "Setja upp"

#: ../UpdateManagerText/UpdateManagerText.py:45
msgid "Building Updates List"
msgstr "Bý til lista yfir uppfærslur"

#: ../UpdateManagerText/UpdateManagerText.py:48
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Ekki var mögulegt að reikna venjulega uppfærslu, vinsamlegast keyrið: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"Örsakavaldur getur verið eftirfarandi:\n"
" * Fyrri uppfærsla kláraði sig ekki sem skildi\n"
" * Vandamál í einhverjum uppsettum hugbúnaði\n"
" * Óformlegir hugbúnaðarpakkar sem ekki er á vegum Ubuntu\n"
" * Venjubundnar breytingar á prufu útgáfu Ubuntu"

#: ../UpdateManagerText/UpdateManagerText.py:117
msgid "Downloading changelog"
msgstr "Sæki breytingarskrá"

#: ../UpdateManager/Core/MyCache.py:108
#, python-format
msgid "Other updates (%s)"
msgstr "Aðrar uppfærslur (%s)"

#: ../UpdateManager/Core/MyCache.py:227
#, python-format
msgid ""
"Changes for the versions:\n"
"%s\n"
"%s\n"
"\n"
msgstr ""
"Breytingar fyrir útgáfurnar:\n"
"%s\n"
"%s\n"
"\n"

#: ../UpdateManager/Core/MyCache.py:229
msgid "This change is not coming from a source that supports changelogs."
msgstr ""

#: ../UpdateManager/Core/MyCache.py:237
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""

#: ../UpdateManager/Core/MyCache.py:242
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Listi yfir breytingar er ekki fáanlegur enn sem komið er.\n"
"\n"
"Notaðu http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"þangað til breytingarnar eru fáanlegar eða reyndu aftur síðar."

#: ../UpdateManager/Core/MyCache.py:248
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Það tókst ekki að sækja lista yfir breytingar. \n"
"Athugaðu með nettenginguna þína."

#: ../UpdateManager/Core/UpdateList.py:50
msgid "Failed to detect distribution"
msgstr "Gat ekki komist að því hvaða útgáfa þetta er"

#: ../UpdateManager/Core/UpdateList.py:51
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr "Villan ‚%s‘ kom upp þegar tölvan var að kanna hvaða kerfi þú notar."

#: ../UpdateManager/Core/UpdateList.py:62
msgid "Important security updates"
msgstr "Mikilvægar öryggisuppfærslur"

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Recommended updates"
msgstr "Uppfærslur sem mælt er með"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Proposed updates"
msgstr "Fyrirhugaðar uppfærslur"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Backports"
msgstr ""

#: ../UpdateManager/Core/UpdateList.py:66
msgid "Distribution updates"
msgstr "Uppfærslur fyrir dreifinguna"

#: ../UpdateManager/Core/UpdateList.py:71
msgid "Other updates"
msgstr "Aðrar uppfærslur"

#: ../UpdateManager/Core/UpdateList.py:80
msgid "Previous selected"
msgstr "Áður valið"

#: ../data/glade/UpdateManager.ui.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>Þú verður að leita að uppfærslum í höndunum</big></b>\n"
"\n"
"Tölvan þín leitar ekki að uppfærslum sjálfkrafa. Þú getur breytt þessu í "
"<i>Hugbúnaðarupptökum</i> undir flipanum <i>Uppfærslur</i>."

#: ../data/glade/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Það er ekki hægt að setja upp allar uppfærslurnar</b></big>"

#: ../data/glade/UpdateManager.ui.h:5
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Nota rafhlöðu</b></big>"

#: ../data/glade/UpdateManager.ui.h:6
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<big><b>Ræsi uppfærslustjórann</b></big>"

#: ../data/glade/UpdateManager.ui.h:7
msgid "Changes"
msgstr "Breytingar"

#: ../data/glade/UpdateManager.ui.h:8
msgid "Chec_k"
msgstr "_Kanna"

#: ../data/glade/UpdateManager.ui.h:9
msgid "Co_ntinue"
msgstr "_Halda áfram"

#: ../data/glade/UpdateManager.ui.h:10
msgid "Description"
msgstr "Lýsing"

#: ../data/glade/UpdateManager.ui.h:11
msgid "Description of update"
msgstr "Lýsing"

#: ../data/glade/UpdateManager.ui.h:13
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""

#: ../data/glade/UpdateManager.ui.h:20
msgid "Settings..."
msgstr "Stillingar..."

#: ../data/glade/UpdateManager.ui.h:21
msgid "Show progress of individual files"
msgstr "Sjá hvernig einstaka skrám gengur"

#: ../data/glade/UpdateManager.ui.h:23
msgid "U_pgrade"
msgstr "U_ppfæra"

#: ../data/glade/UpdateManager.ui.h:24 ../data/update-manager.desktop.in.h:3
#: ../data/update-manager-hildon.desktop.in.h:3
msgid "Update Manager"
msgstr "Uppfærslustjóri"

#: ../data/glade/UpdateManager.ui.h:25
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr ""
"Tölvan þín fær rafmagn frá rafhlöðu. Ertu viss um að þú viljir halda áfram?"

#: ../data/glade/UpdateManager.ui.h:26
msgid "_Hide this information in the future"
msgstr "_Fela þessar upplýsingar í framtíðinni"

#: ../data/glade/UpdateManager.ui.h:27
msgid "_Install Updates"
msgstr "_Setja upp uppfærslur"

#: ../data/glade/UpdateManager.ui.h:28
msgid "_Partial Upgrade"
msgstr "_Hlutauppfærsla"

#: ../data/glade/UpdateManager.ui.h:29
msgid "_Upgrade"
msgstr "_Uppfæra"

#: ../data/glade/UpdateManager.ui.h:30
msgid "updates"
msgstr "uppfærslur"

#: ../data/update-manager.desktop.in.h:1
#: ../data/update-manager-hildon.desktop.in.h:1
msgid "Show and install available updates"
msgstr "Sýna og setja upp uppfærslur"

#: ../data/update-manager.desktop.in.h:2
#: ../data/update-manager-hildon.desktop.in.h:2
msgid "Software Updates"
msgstr "Hugbúnaðaruppsetningar"

#: ../data/update-manager.schemas.in.h:1
msgid "*deprecated* Check for new distribution releases"
msgstr "*óæskilegt* Leita að nýjum útgáfum"

#: ../data/update-manager.schemas.in.h:2
msgid "Auto close the install window after successful install"
msgstr "Loka glugganum sjálfkrafa ef uppfærslan heppnast"

#: ../data/update-manager.schemas.in.h:3
msgid "First run welcome message"
msgstr ""

#: ../data/update-manager.schemas.in.h:4
msgid ""
"If automatic checking for updates is disabled, you have to reload the "
"channel list manually. This option allows to hide the reminder shown in this "
"case."
msgstr ""

#: ../data/update-manager.schemas.in.h:5
msgid "If this key is set a first run welcome message will be presented."
msgstr ""

#: ../data/update-manager.schemas.in.h:6
msgid ""
"If this key is set the install window will be automatically closed on "
"successful installation."
msgstr ""

#: ../data/update-manager.schemas.in.h:7
msgid ""
"If this key is set the main update list window will show version information "
"(from version to version)."
msgstr ""

#: ../data/update-manager.schemas.in.h:8
msgid "Remind to reload the channel list"
msgstr "Minna á að endurhlaða listann yfir rásir"

#: ../data/update-manager.schemas.in.h:9
msgid "Show details of an update"
msgstr "Birta smáatriði uppfærslna"

#: ../data/update-manager.schemas.in.h:10
msgid "Show version in update list"
msgstr "Sýna útgáfu í uppfærslulista"

#: ../data/update-manager.schemas.in.h:11
msgid "Stores the size of the update-manager dialog"
msgstr "Geymir stærðina á svarglugga uppfærslustjórans"

#: ../data/update-manager.schemas.in.h:12
msgid ""
"Stores the state of the expander that contains the list of changes and the "
"description"
msgstr ""

#: ../data/update-manager.schemas.in.h:13
msgid "The window size"
msgstr "Gluggastærð"

#: ../data/update-manager.schemas.in.h:14
msgid ""
"This key is deprecated in favor of the file /etc/update-manager/release-"
"upgrades Check automatically if a new version of the current distribution is "
"available and offer to upgrade (if possible)."
msgstr ""

#: ../update-manager:56
msgid "Show version and exit"
msgstr "Sýna útgáfu og hætta"

#: ../update-manager:59
msgid "Directory that contains the data files"
msgstr ""

#: ../update-manager:62
msgid "Check if a new Ubuntu release is available"
msgstr "Athuga hvort til sé ný útgáfa af Ubuntu"

#: ../update-manager:65 ../do-release-upgrade:23 ../check-new-release:24
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""

#: ../update-manager:69
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr ""

#: ../update-manager:76
msgid "Do not focus on map when starting"
msgstr ""

#: ../update-manager:79
msgid "Try to run a dist-upgrade"
msgstr "Reyndu að keyra drefingaruppfærslu"

#: ../update-manager:83 ../do-release-upgrade:39
msgid "Test upgrade with a sandbox aufs overlay"
msgstr ""

#: ../update-manager:103
msgid "Running partial upgrade"
msgstr "Keyra hlutauppfærslu"

#: ../do-release-upgrade:27 ../check-new-release:28
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""

#: ../do-release-upgrade:31
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""

#: ../do-release-upgrade:37
msgid "Run the specified frontend"
msgstr ""

#: ../do-release-upgrade:43
msgid "Checking for a new ubuntu release"
msgstr "Leita að nýrri Ubuntu útgáfu"

#: ../do-release-upgrade:50
msgid "No new release found"
msgstr "Engar nýjar útgáfur fundust"

#: ../check-new-release:32
#, c-format
msgid ""
"Do not output anything, just return %s if there is a new release and %s if "
"there is none."
msgstr ""
"Ekki hafa neitt frálag, skila bara %s ef það er ný útgáfa og %s ef engin ný "
"útgáfa finnst."

#: ../check-new-release:46
#, c-format
msgid "New release '%s' available."
msgstr "Nýja útgáfan ‚%s‘ fannst."

#: ../check-new-release:47
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Keyrðu ‚do-release-upgrade‘ til að hefja uppfærsluna."

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr "Aðferð sem er ekki notuð: %s"

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr "Skrá á diski"

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr ".deb pakki"

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr "Setja upp pakkann sem vantar."

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr "Það ætti að setja upp pakkann %s."

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr ""

#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr ""

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr ""

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr ""

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"Fjarlægja lilo þar sem grub er til staðar.(See bug #314004 for details.)"

#~ msgid "Upgrade to the latest version of Ubuntu"
#~ msgstr "Uppfæra í nýjustu útgáfu af Ubuntu"

#~ msgid "Verfication failed"
#~ msgstr "Sannprófun mistókst"
